pom.xml

```diff
@@ -38,7 +38,7 @@
 
         <github.url>scm:git:git@github.com:dockstore/dockstore-support.git</github.url>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <dockstore-core.version>1.15.0-alpha.5</dockstore-core.version>
+        <dockstore-core.version>1.15.0-SNAPSHOT</dockstore-core.version>
```

Comment:
I will update this to a tag when https://github.com/dockstore/dockstore/pull/5738 is merged